python/MaterialXTest/genshader.py

```diff
@@ -11,7 +11,7 @@ class TestGenShader(unittest.TestCase):
     def test_ShaderInterface(self):
         doc = mx.createDocument()
 
-        libraryFolders = ["stdlib"]
+        libraryFolders = ["targets", "stdlib"]
```

Comment:
As mentioned above the new "targets" library is loaded in, in order to have all default targetdefs loaded and supported.